src/ImageSharp/Processing/Processors/Convolution/BokehBlurProcessor{TPixel}.cs

```diff
@@ -0,0 +1,531 @@
+                            PixelOperations<TPixel>.Instance.ToVector4(configuration, targetRowSpan.Slice(0, length), vectorSpan, PixelConversionModifiers.Premultiply);
+                            ref Vector4 baseRef = ref MemoryMarshal.GetReference(vectorSpan);
+
+                            for (int x = 0; x < width; x++)
+                            {
+                                ref Vector4 v = ref Unsafe.Add(ref baseRef, x);
+                                v.X = (float)Math.Pow(v.X, gamma);
+                                v.Y = (float)Math.Pow(v.Y, gamma);
+                                v.Z = (float)Math.Pow(v.Z, gamma);
+                            }
+
+                            PixelOperations<TPixel>.Instance.FromVector4Destructive(configuration, vectorSpan.Slice(0, length), targetRowSpan);
+                        }
+                    });
+        }
+
+        /// <summary>
+        /// Applies the inverse gamma correction/highlight pass, and converts the input <see cref="Vector4"/> buffer into pixel values.
+        /// </summary>
+        /// <param name="targetPixels">The target pixels to apply the process to.</param>
+        /// <param name="sourceValues">The source <see cref="Vector4"/> values. Cannot be null.</param>
+        /// <param name="sourceRectangle">
+        /// The <see cref="Rectangle"/> structure that specifies the portion of the image object to draw.
+        /// </param>
+        /// <param name="configuration">The <see cref="Configuration"/></param>
+        private void ApplyInverseGammaExposure(
+            Buffer2D<TPixel> targetPixels,
+            Buffer2D<Vector4> sourceValues,
+            Rectangle sourceRectangle,
+            Configuration configuration)
+        {
+            int startY = sourceRectangle.Y;
+            int endY = sourceRectangle.Bottom;
+            int startX = sourceRectangle.X;
+            int endX = sourceRectangle.Right;
+
+            var workingRectangle = Rectangle.FromLTRB(startX, startY, endX, endY);
+            int width = workingRectangle.Width;
+            float expGamma = 1 / this.gamma;
+
+            ParallelHelper.IterateRows(
+                workingRectangle,
+                configuration,
+                rows =>
+                    {
+                        for (int y = rows.Min; y < rows.Max; y++)
+                        {
+                            Span<Vector4> targetRowSpan = sourceValues.GetRowSpan(y).Slice(startX);
+                            Span<TPixel> targetPixelSpan = targetPixels.GetRowSpan(y).Slice(startX);
+                            ref Vector4 baseRef = ref MemoryMarshal.GetReference(targetRowSpan);
+
+                            for (int x = 0; x < width; x++)
+                            {
+                                ref Vector4 v = ref Unsafe.Add(ref baseRef, x);
+                                v.X = (float)Math.Pow(v.X.Clamp(0, float.PositiveInfinity), expGamma);
```

Comment:
`Vector4.Clamp` with Hi/Low defined outside the loop.
Use `MathF.Pow`

Reply:
Done in https://github.com/SixLabors/ImageSharp/pull/842/commits/04984cd65006b8794fcef38564ef1414654eecb5 and https://github.com/SixLabors/ImageSharp/pull/842/commits/a70518bcdad126be73aa97b5a1398b4af4c52bce.